docs/features/software-catalog/descriptor-format.md

```diff
@@ -444,6 +446,22 @@ Apart from being a string, the software catalog leaves the format of this field
 open to implementers to choose. Most commonly, it is set to the ID or email of a
 group of people in an organizational structure.
 
+### `spec.system` [optional]
+
+Links the system that the component belongs to, e.g. `artist-engagement-portal`.
+This field is optional.
+
+The software catalog expects a single string that references the name of an
+entity of the `kind` `System`.
```

Comment:
I think this is an opportunity to settle on a nice (kinda copy-pasteable) form of [this](https://github.com/backstage/backstage/blob/master/docs/features/software-catalog/descriptor-format.md#specmemberof-required).

Something like 

```
The value of this field is an
[entity reference](https://backstage.io/docs/features/software-catalog/references),
with the default kind `System` and the default namespace equal to the same
namespace as this entity. Only `System` entities may be referenced. Most commonly,
this field points to a system in the same namespace, so in those cases it is
sufficient to enter only the `metadata.name` field of that system. The value entered
here results in a `partOf`/`hasPart` [relation](<link to that exact part of the well known relations doc>)
to the target.
```

Or maybe we could have a more succinct table form describing all of these refs?

| <!-- -->  | <!-- --> |
| - | - |
| Default kind | `System` (only) |
| Default namespace | Same as this entity, typically `default` |
| Generated relation type | `partOf`, and reverse `hasPart` |

Can't come up with what to put in the (required) table header though and it looks a bit funny when left empty. Maybe twist it, but that doesn't look great either.

| Default [kind](https://github.com/backstage/backstage/blob/master/docs/features/software-catalog/descriptor-format.md#apiversion-and-kind-required) | Default [namespace](https://github.com/backstage/backstage/blob/master/docs/features/software-catalog/descriptor-format.md#namespace-optional) | Generated [relation](https://github.com/backstage/backstage/blob/master/docs/features/software-catalog/well-known-relations.md) type |
| - | - | - |
| `System` (only) | Same as this entity, typically `default` | `partOf`, and reverse `hasPart` |

It's nice and friendly to use a lot of those cross links, I think.